SUPPORT.md

```diff
@@ -0,0 +1,14 @@
+# Support
+
+## How to file issues and get help
+
+This project uses GitHub Issues to track bugs and feature requests. Please search the [existing issues][github-issues] before filing new issues to avoid duplicates. For new issues, file your bug or feature request as a new Issue.
+
+For help and questions about using this project, please open a new GitHub issue or visit the [#aks-engine-users][aks-engine-users-slack] Kubernetes Slack channel. The AKS Engine project maintainers will respond to the best of their abilities.
+
+## Microsoft Support Policy
+
+Support for this project is limited to the resources listed above.
+
+[aks-engine-users-slack]: https://kubernetes.slack.com/archives/CU3N85WJK
```

Comment:
Can you move these 2 points under to under the 'For help and questions' section?

Reply:
👍 Done.